Dockerfile

```diff
@@ -132,8 +133,8 @@ EXPOSE 80 443
 
 LABEL name="manageiq" \
           vendor="ManageIQ" \
-          version="Capablanca" \
-          release="latest" \
+          version="Darga" \
+          release=${REF} \
```

Comment:
@Fryguy @bdunne these labels actually behave like in RPM standard, one could say version="Capablanca/Darga" , release="-2", I think it makes sense to give the author/builder freedom to handle these as needed via build ARGs with some good defaults. Also it would be nice if the builder could pick a MIQ repo/branch where to build from via a  build-arg. The name/release/version labels are more straightforward downstream, name="cfme", version=5.6 , release="-4" , image registries can utilize this information.